docs-utils/update-images.js

```diff
@@ -22,26 +23,32 @@ const getExtension = (url) => {
 }
 
 const updateImages = async (filepath) => {
+  console.log("Running update images")
+  console.log(filepath)
   const content = fs.readFileSync(filepath, 'utf-8')
   const slug = frontmatter(content).attributes.slug
   const locale = frontmatter(content).attributes.locale
   
   const images = []
+  let imageIndex = 1
   const replaceURL = (match, extra, url) => {
     const isMarkdownBlock = match.startsWith('![')
     if (url.startsWith(baseURL)) return match
 
     let newURL = ''
     if (url.startsWith('http://') || url.startsWith('https://')) {
       const ext = getExtension(url)
-      const filename = `${slug}-${images.length}.${ext}`
-  
+      
+      const newfilepath = `${filepath.split('.')[0]}_${imageIndex}.${ext}`
+      console.log(newfilepath)
```

Comment:
I think we could get rid of this log here. It has fulfilled its debugging purpose already.